pkg/syncer/types.go

```diff
@@ -100,8 +100,11 @@ type (
 )
 
 type metadataSyncInformer struct {
-	clusterFlavor      cnstypes.CnsClusterFlavor
-	volumeManager      volumes.Manager
+	clusterFlavor cnstypes.CnsClusterFlavor
+	volumeManager volumes.Manager
+	// map of VC Host to Volume Manager
+	// Use this for Vanilla Multi vCenter Topology feature
+	volumeManagers     map[string]volumes.Manager
 	host               string
 	cnsOperatorClient  client.Client
 	supervisorClient   clientset.Interface
```

Comment:
Do we also want to initialise configInfo for each VC?

I think we can skip it as we can get the same info from volumeManagers["host-name"].virtualCenter.Config

Reply:
configInfo is common and contains both VC detail.